translate/src/modules/editor/components/EditorSettings.tsx

```diff
@@ -43,21 +45,23 @@ export function EditorSettingsDialog({
         </li>
       </Localized>
 
-      <Localized
-        id='editor-EditorSettings--force-suggestions'
-        attrs={{ title: true }}
-        elems={{ glyph: <i className='fa fa-fw' /> }}
-      >
-        <li
-          className={
-            'check-box' + (settings.forceSuggestions ? ' enabled' : '')
-          }
-          title='Save suggestions instead of translations'
-          onClick={() => toggleSetting('forceSuggestions')}
+      {isTranslator ? (
+        <Localized
+          id='editor-EditorSettings--force-suggestions'
+          attrs={{ title: true }}
+          elems={{ glyph: <i className='fa fa-fw' /> }}
         >
-          {'<glyph></glyph>Make Suggestions'}
-        </li>
-      </Localized>
+          <li
+            className={
+              'check-box' + (settings.forceSuggestions ? ' enabled' : '')
+            }
+            title='Save suggestions instead of translations'
+            onClick={() => toggleSetting('forceSuggestions')}
+          >
+            {'<glyph></glyph>Make Suggestions'}
```

Comment:
While we're in here, we should also fix the capitalization to match the /settings page (also in the .ftl files):
https://github.com/mozilla/pontoon/commit/d8696c9deb54835272b6219ab97fd8a9c9f70802#diff-a8bb92e441421dd7d5626b58519e1827a26d8635c8884cfd17dd0bcd4303f68fR120